api/src/main/java/io/terrakube/api/plugin/scheduler/job/tcl/executor/persistent/PersistentExecutorService.java

```diff
@@ -61,7 +60,7 @@ public void send(Job job, ExecutorContext executorContext) throws ExecutionExcep
                     .retrieve()
                     .toEntity(ExecutorContext.class)
                     .block();
-        } catch (URISyntaxException | WebClientResponseException ex) {
+        } catch (Exception ex) {
             throw new ExecutionException(ex);
         }
```

Comment:
Communication errors would tend to be runtime errors which we also want to convert to checked exceptions.